sdk/storage/azure-storage-files-shares/swagger/README.md

```diff
@@ -219,6 +219,8 @@ directive:
       $.ShareEnabledProtocols["enum"] = ["Smb", "Nfs"];
       $.ShareEnabledProtocols["x-ms-enum"] = {"name": "ShareProtocols", "modelAsString": false};
       $.ShareEnabledProtocols["x-ms-enum"]["values"] = [{"value": "SMB", "name": "Smb"},{"value": "NFS", "name": "Nfs"}];
+      delete $.FileChangeTime.format;
+      $.FileLastWriteTimeMode["x-ms-enum"]["values"] = [{"value": "now", "name": "Now"},{"value": "preserve", "name": "Preserve"}];
```

Comment:
Why do we have to modify the swagger in this case?
Is the swagger wrong, and do other languages behave the same way?

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Files.Shares/src/autorest.md#formats-arent-required

Reply:
Other languages also do this.